### ISSUE TITLE
Integrating tinasaurus with github

### ISSUE THREAD

**Issue by ararTP** [tinacms/tinasaurus#20]
Is it possible to use Github and self hosting tinasaurus server? 
I want to be able to integrate it with Github, like in [tina-self-hosted](https://github.com/tinacms/tina-self-hosted-demo) , but there is no `database.ts` file / any other docs about it.
How can I do it with a local Tina server , without Tina Cloud? 
(My end goal is to have the content also updating in my github after editing)
Thanks

**Comment by wicksipedia** [comment]
Yes, you can self-host Tina and integrate it with GitHub without using Tina Cloud.

Start with the official self-hosting guide:  
https://tina.io/docs/self-hosted/overview/

It explains how to run a local `@tinacms/cms` server, configure `database.ts`, and push content updates to GitHub using a local auth setup.

Note: Tinasaurus doesn’t include `database.ts` by default—you’ll need to add one following the guide.

You can hook it up to GitHub by using it's provider package:  
https://github.com/tinacms/tinacms/tree/main/packages/tinacms-gitprovider-github

If you need help getting it running, feel free to reach out in our Discord:  
https://discord.com/invite/zumN63Ybpf
